smartsim/ml/data.py

```diff
@@ -29,8 +29,6 @@
 from os import environ
 
 import numpy as np
-from smartredis import Client, Dataset
-from smartredis.error import RedisReplyError
```

Comment:
Ditto "shouldn't matter, but symbols are no longer defined and being used" comment here. Consider setting up some mocks or removing the usage to get past the CI.

Reply:
added mocks